health/notifications/alarm-notify.sh.in

```diff
@@ -2321,14 +2321,22 @@ EOF
 # Opsgenie sender
 
 send_opsgenie() {
-  local payload httpcode oldv currv
+  local payload httpcode oldv currv priority
   [ "${SEND_OPSGENIE}" != "YES" ] && return 1
 
   if [ -z "${OPSGENIE_API_KEY}" ] ; then
     info "Can't send Opsgenie notification, because OPSGENIE_API_KEY is not defined"
     return 1
   fi
 
+  # Priority for OpsGenie alert
+  case "${status}" in
+  CRITICAL) priority="P5" ;; # Critical is P5
+  WARNING) priority="P3" ;; # Warning is P3
+  CLEAR) priority="P1" ;; # Clear is P1
+  *) priority="P3" ;; # OpsGenie's default alert level is P3
+  esac
```

Comment:
I see that the [importance level decreases from P1 to P5](https://support.atlassian.com/opsgenie/docs/what-is-the-priority-level-of-integration/).

According to [Update alert priority level](https://support.atlassian.com/opsgenie/docs/update-alert-priority-level/):

- P1 is Critical.
- P2 is High
- P3 is Moderate.
- P4 is Low.
- P5 is informational.

Reply:
damn, that's embarassing. ive force pushed the branch with the fix. although happy if you guys implement yourselves as this PR is more of a suggestion/easier to draft than an issue

Reply:
No worries, @OliverNChalk. Your changes look good, thanks 👍